pkg/httpserver/server_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/trustbloc/sidetree-core-go/pkg/restapi/helper"
```

Comment:
wrong location

Reply:
Done. I noticed that I was still referencing model in one case so I fixed that too.